pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -35,26 +35,21 @@ display(tile) # IPython.display function
 
 ## DataFrame Samples
 
-Within an IPython or Jupyter interpreter, a Pandas DataFrame containing a column of _tiles_ will be rendered as the samples discussed above. Simply import the `rf_ipython` submodule to enable enhanced HTML rendering of a Pandas DataFrame.
+Within an IPython or Jupyter interpreter, a Spark DataFrame containing a column of _tiles_ will be rendered as the samples discussed above. Simply import the `rf_ipython` submodule to enable enhanced HTML rendering of a Spark DataFrame.
```

Comment:
It is true for either Spark or Pandas. We should mention that here. And "The rendering for Spark DataFrames will be more efficient than converting a Spark DataFrame to Pandas.